### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by ljy199712** [brandleyzhou/DIFFNet#24]
Thanks for your working.
Here are something detials i want 2 ask you . Here are my torch
torch                     1.7.1+cu110              torchaudio                0.7.2                    torchsummary              1.5.1                    
torchvision               0.8.2+cu110 
I found when i  set the initial learning rate as 10−4 for the first 14 epochs and then 10−5 for last 5 epochs ,my experimental results are very different from yours  . Is it the reason for different PyTorch versions？Or  my training process wrong?

**Comment by brandleyzhou** [comment]
you are right, the PyTorch version effects. please use torch 1.1.0 for training

**Comment by ljy199712** [comment]
Thanks for your time, a error in loading weight when i change the PyTorch versions.
![微信图片_20220829102246](https://user-images.githubusercontent.com/88530858/187111058-83896536-f989-499a-bc99-3954ccd928aa.jpg)

**Comment by brandleyzhou** [comment]
It seems you need to manually download the pretrained hrnet18. please refer to this issue https://github.com/brandleyzhou/DIFFNet/issues/7
